msg/ko_KR.utf8/cubrid.msg

```diff
@@ -1443,7 +1443,11 @@ $ LOADDB
 1357 SQL 문자열을 와이드 문자열로 변환 실패.
 1358 영향을 받은 행 수를 알 수 없음.
 
-1359 마지막 에러
+1359 Java VM crashed: %1$s
+
+1360 Java VM 에 장애가 발생함. : %1$s
```

Comment:
Unlike msg files in other languages, there are two 1360. Please confirm it.

Reply:
fixed